packages/ui-components/src/components/search.tsx

```diff
@@ -172,23 +172,26 @@ export const FilterBox = (props: IFilterBoxProps): JSX.Element => {
   /**
    * Handler for search input changes.
    */
-  const handleChange = (e: React.FormEvent<HTMLElement>) => {
-    const target = e.target as HTMLInputElement;
-    setFilter(target.value);
-    props.updateFilter(
-      updateFilterFunction(
-        target.value,
-        props.useFuzzyFilter,
-        props.caseSensitive
-      ),
-      target.value
-    );
-  };
+  const handleChange = useCallback(
+    (e: Event) => {
+      const target = e.target as HTMLInputElement;
+      setFilter(target.value);
+      props.updateFilter(
+        updateFilterFunction(
+          target.value,
+          props.useFuzzyFilter,
+          props.caseSensitive
+        ),
+        target.value
+      );
+    },
+    [props.updateFilter, props.useFuzzyFilter, props.caseSensitive]
+  );
 
   return (
     <Search
       className="jp-FilterBox"
-      ref={props.inputRef}
+      ref={props.inputRef as any}
```

Comment:
Can we replace `any` with the correct typing here?

Reply:
The webcomponent does not exactly matches a HTMLInputElement. But I can improve it a bit.